modules/mono/editor/editor_internal_calls.cpp

```diff
@@ -152,7 +152,7 @@ bool godot_icall_Internal_ScriptEditorEdit(Resource *p_resource, int32_t p_line,
 }
 
 void godot_icall_Internal_EditorNodeShowScriptScreen() {
-	EditorNode::get_singleton()->call("_editor_select", EditorNode::EDITOR_SCRIPT);
+	EditorNode::get_singleton()->_editor_select(EditorNode::EDITOR_SCRIPT);
```

Comment:
Since this one is actually used as a public method, it should not have the underscore prefix and the comment in EditorNode doesn't apply to it.